packages/integrations/readme.md

```diff
@@ -6,3 +6,48 @@ A collection of Irving-friendly components for getting you started with common t
 * `<GoogleAnalytics />` - Render the required JS snippet for your GA account.
 * `<GoogleTagManager />` - Render the required JS snippet for your GTM account.
 * `<Parsely />` - Render the required JS for sending data to Parse.ly
+* `<YoastSchema />` - Render the required JS for Yoast.
+---
+### `<IntegrationsManager />`
+The integrations manager is a component that consumes a `irving/integrations` configuration retrieved from the REST API that dynamically configures, stores props for, and renders corresponding integrations based on the keys set in the configuration object.
+
+### Configuration
+Configuring the integrations manager is a simple process. First, add the integrations manager component to the irving componentMap. Example:
+```js javascript
+import { IntegrationsManager } from '@irvingjs/integrations';
```

Comment:
Could this export be a default?

Reply:
I don't think that would fit the established pattern. Right now the packages export the`config` as default and the components as individual named exports. Open to @ostowe's opinion, though.